docs/content/docs/deployment/elastic_scaling.md

```diff
@@ -230,6 +230,22 @@ In addition, there are several related configuration options that may need adjus
   - Set `parallelism.default: -1`.
   - Don't call `setParallelism()` on `ExecutionEnvironment`.
 
+#### Enable dynamic parallelism inference support for Sources
+If your job uses a custom {{< gh_link file="/flink-core/src/main/java/org/apache/flink/api/connector/source/Source.java" name="Source" >}},
```

Comment:
If your job uses a custom `Source`, you need to implement the interface `DynamicParallelismInference`...
->
New `Source` can implement `DynamicParallelismInference` to enable dynamic parallelism inference.

I think it's better to state it as below. Because dynamic parallelism inference is not a requirement but just an option for users.